contrib/verify-commits/trusted-keys

```diff
@@ -0,0 +1 @@
+1E7DD3547F244C757223FAC8FC6434FC3A31E956
```

Comment:
my GPG key fingerprint is 32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC

-----BEGIN PGP MESSAGE-----
Version: GnuPG v1

owEBeQKG/ZANAwACASnUvLZBb1PsActJYgBXvWG4bXkgR1BHIGtleSBmaW5nZXJw
cmludCBpcyAzMkVFNUM0QzNGQTE1Q0NBREI0NkFCRTUyOUQ0QkNCNjQxNkY1M0VD
CokCHAQAAQIABgUCV71huAAKCRAp1Ly2QW9T7P7sD/oDyR6DmdwqPcLSaxJvwvGZ
2CjQeO88IVafLvPG193wvhFNbg62cDk6f8jvAaqJvedIMLjNL1uWAVlwcINPmccJ
8VhjVX5RjpLOHJszlW2FEryY7Pax47htofmyq4avMxdJJfKv/chCwJJ/oxzFQ7TJ
UIKtttaTy9sbb0DPYOWqJyAhyErU1eBP3oOpixr1+H+kRREJPjsTfDEoBiNsgdiK
xnziB9i0+NVYAjUWHSbusrkAIQz6a7CSspChslsFALTxtWIgWDcrZXjYeVGNShPG
wSBYTpyr7XTTSB5Q+eTTO2cbJP3Rcv4RKeFy5aUsdDiMga4JG8mlvLq2rGGvzAcd
qo8IZ+Rxw3um7YpcyjNKSKXCwDisB5Rgnbf97O8JSoT2NAN9hRkK4DMoAoDR+3Hl
WlRSDXd4cvvhmU5wt6oF7hL8RfPVfdvX55uhW1EP/mbXJlAi+8Q3S2XvbZoP7M49
ZsksUGjHIVhoxv83Csz6RZu1wh+EnT97wBX9Q+jGuOJgY56RJycvMcWFSgWaGsgR
Dvpe7LCP2XU9DWVT/tsxbln4o8c6qLN/Aq/53SNfB4z0Wjmzz/hLIHaut3ubiDXh
veWu/s1MwU7bUTMP5GqIP5e/w9ChTUR8Ui/UVco31gZlv1iX1Q7gqVxZLq22RT1u
odsqAXw+kmay5bKJqFZD0A==
=4+7b
-----END PGP MESSAGE-----